indigo_app/templates/indigo_api/work_form.html

```diff
@@ -338,6 +338,21 @@ <h5 class="card-header">Repeal</h5>
       </div>
     </div>
     {% endblock work-form-repeal %}
+    <div class="form-row">
```

Comment:
Please put this inside a `.card` with an appropriate header, so that it matches the layout of the rest of the page.

An idea would be to put this together with the Assent date, since that's combined with commencement which is pretty complex. So Commencement would be on its own, and "Key dates" would have Assent an As-at date. Thoughts?

Reply:
yeah that makes sense! lmk what you think of the screenshot